internal/v4/list_test.go

```diff
@@ -260,51 +259,51 @@ func (s *ListSuite) TestMetadataFields(c *gc.C) {
 	}
 }
 
-func (s *ListSuite) TestListIncludeError(c *gc.C) {
-	// Perform a list for all charms, including the
-	// manifest, which will try to retrieve all charm
-	// blobs.
-	rec := httptesting.DoRequest(c, httptesting.DoRequestParams{
-		Handler: s.srv,
-		URL:     storeURL("list?type=charm&include=manifest"),
-	})
-	c.Assert(rec.Code, gc.Equals, http.StatusOK)
-	var resp params.ListResponse
-	err := json.Unmarshal(rec.Body.Bytes(), &resp)
-	// cs:riak will not be found because it is not visible to
-	// "everyone".
-	c.Assert(resp.Results, gc.HasLen, len(exportListTestCharms)-1)
-
-	// Now remove one of the blobs. The list should still
-	// work, but only return a single result.
-	entity, err := s.store.FindEntity(newResolvedURL("~charmers/precise/wordpress-23", 23), nil)
-	c.Assert(err, gc.Equals, nil)
-	err = s.store.BlobStore.Remove(entity.BlobName, nil)
-	c.Assert(err, gc.Equals, nil)
-
-	// Now list again - we should get one result less
-	// (and the error will be logged).
-
-	// Register a logger that so that we can check the logging output.
-	// It will be automatically removed later because IsolatedMgoESSuite
-	// uses LoggingSuite.
-	var tw loggo.TestWriter
-	err = loggo.RegisterWriter("test-log", &tw)
-	c.Assert(err, gc.Equals, nil)
-
-	rec = httptesting.DoRequest(c, httptesting.DoRequestParams{
-		Handler: s.srv,
-		URL:     storeURL("list?type=charm&include=manifest"),
-	})
-	c.Assert(rec.Code, gc.Equals, http.StatusOK)
-	resp = params.ListResponse{}
-	err = json.Unmarshal(rec.Body.Bytes(), &resp)
-	// cs:riak will not be found because it is not visible to "everyone".
-	// cs:wordpress will not be found because it has no manifest.
-	c.Assert(resp.Results, gc.HasLen, len(exportListTestCharms)-2)
-
-	c.Assert(tw.Log(), jc.LogMatches, []string{"cannot retrieve metadata for cs:precise/wordpress-23: cannot open archive data for cs:precise/wordpress-23: .*"})
-}
+//func (s *ListSuite) TestListIncludeError(c *gc.C) {
```

Comment:
Added comment. Skip isn't a good option because the code refers to fields that no longer exist.